plasmapy/physics/tests/test_parameters.py

```diff
@@ -523,26 +516,27 @@ def test_gyroradius():
         gyroradius(1.1 * u.T, T_i=1.2 * u.K, Vperp=1 * u.m / u.s)
 
     with pytest.raises(ValueError):
-        gyroradius(1.1 * u.T, 1.2 * u.K, 1.1 * u.m)
+        gyroradius(1.1 * u.T, T_i=1.2 * u.K, Vperp=1.1 * u.m)
 
-    assert gyroradius(B, T_i, particle="p").unit == u.m
+    assert gyroradius(B, T_i=T_i, particle="p").unit == u.m
 
-    assert gyroradius(B, 25 * u.m / u.s, particle="p").unit == u.m
+    assert gyroradius(B, Vperp=25 * u.m / u.s, particle="p").unit == u.m
 
     # Case when Z=1 is assumed
-    assert gyroradius(B, T_i, particle='p') == \
-        gyroradius(B, T_i, particle='H-1')
-
-    assert gyroradius(T_i, B, particle="p") == gyroradius(B, T_i, particle="p")
-
-    assert gyroradius(V, B, particle="p") == gyroradius(B, V, particle="p")
+    assert np.isclose(gyroradius(B, T_i=T_i, particle='p'),
+                      gyroradius(B, T_i=T_i, particle='H+'),
+                      atol=1e-6*u.m)
 
-    assert gyroradius(B, V, particle="p") == gyroradius(B, -V, particle="p")
+    assert gyroradius(B,
```

Comment:
Split into 3 lines so that it is less horrible to look at.
```
gyroPos = gyroradius(B, Vperp=V, particle="p")
gyroNeg = gyroradius(B, Vperp=-V, particle="p")
assert gyroPos == gyroNeg
```